source/_components/nissan_leaf.markdown

```diff
@@ -0,0 +1,126 @@
+---
+layout: page
+title: "Nissan Leaf"
+description: "Instructions for how to integrate Nissan Leaf(s) into Home Assistant."
+date: 2019-01-05 07:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: nissan.png
+ha_category: Car
+ha_release: 0.88
```

Comment:
0.89

Reply:
@fabaff has already [fix this](https://github.com/home-assistant/home-assistant.io/commit/cebb49b6e9217c0e3a5039412355f1d51de34870) 😉